code/modules/mob/living/simple_animal/simple_animal_vr.dm

```diff
@@ -92,20 +96,48 @@
 /mob/living/simple_animal/PunchTarget()
 	ai_log("vr/PunchTarget() [target_mob]", 3)
 
-	// For things we don't want to eat, call the sideways "parent" to do normal punching
-	if(!vore_active || !will_eat(target_mob))
+	// If we're not hungry, call the sideways "parent" to do normal punching
+	if(!vore_active)
+		return ..()
+
+	// If target is standing we might pounce and knock them down instead of attacking
+	var/pouncechance = CanPounceTarget()
+	if(pouncechance)
+		return PounceTarget(pouncechance)
+
+	// We're not attempting a pounce, if they're down or we can eat standing, do it as long as they're edible. Otherwise, hit normally.
+	if(will_eat(target_mob) && (!target_mob.canmove || vore_standing_too))
+		return EatTarget()
+	else
 		return ..()
 
-	// If target is standing we might pounce and eat them
-	if(target_mob.canmove && prob(vore_pounce_chance))
+/mob/living/simple_animal/proc/CanPounceTarget() //returns either FALSE or a %chance of success
+	if(!target_mob.canmove || issilicon(target_mob) || world.time < vore_pounce_cooldown) //eliminate situations where pouncing CANNOT happen
+		return FALSE
+	if(!prob(vore_pounce_chance)) //mob doesn't want to pounce
+		return FALSE
+	if(will_eat(target_mob) && vore_standing_too) //100% chance of hitting people we can eat on the spot
+		return 100
+	var/TargetHealthPercent = (target_mob.health/target_mob.maxHealth)*100 //now we start looking at the target itself
```

Comment:
There are procs to get max and current health, that take into account things like modifiers. Important to use those and not the vars to take those into account if they're buffed by whatever.